docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -47,17 +53,16 @@ cluster.initial_master_nodes:
   - master-c
 --------------------------------------------------
 
-You can use a mix of IP addresses and node names too. If there is more than one
-Elasticsearch node with the same IP address then the transport port must also
-be given to specify exactly which node is meant:
+You can also use IP addresses, or IP addresses and ports, or even a mix of IP
```

Comment:
While node names are generally preferred, you can also use IP addresses, or IP addresses and ports, or even a mix of IP addresses and node names:

Reply:
Ok, qualified this with `If it is not possible to use the names of the nodes...`.